libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/FacebookAdapter.cs

```diff
@@ -241,16 +202,16 @@ public async Task ProcessAsync(HttpRequest request, HttpResponse response, IBot
 
                                 // indicate that this message was received in standby mode rather than normal mode.
                                 (message as dynamic).standby = true;
-                                await this.ProcessSingleMessageAsync(message, bot.OnTurnAsync);
+                                await ProcessSingleMessageAsync(message, bot.OnTurnAsync, cancellationToken).ConfigureAwait(false);
                             }
                         }
                     }
 
                     // send code 200
                     response.StatusCode = 200;
                     response.ContentType = "text/plain";
-                    string text = string.Empty;
-                    await response.WriteAsync(text);
+                    var text = string.Empty;
```

Comment:
Why initialize an empty string? Could be simplified as `await response.WriteAsync(string.Empty).ConfigureAwait(false);` if it is not going to be used afterwards